kuma/wiki/apps.py

```diff
@@ -146,4 +152,4 @@ def on_document_spam_attempt_save(self, sender, instance, **kwargs):
         body = render_to_string('wiki/email/spam.ltxt',
                                 {'spam_attempt': instance})
         send_mail(subject, body, settings.DEFAULT_FROM_EMAIL,
-                  ['mdn-spam-watch@mozilla.com'])
+                  [config.EMAIL_LIST_FOR_FIRST_EDITS])
```

Comment:
Huh, didn't see that, I wonder if it would make sense to rename the config variable to better express what this is used for (e.g. `EMAIL_LIST_SPAM_WATCH`).